runtime/doc/api.txt

```diff
@@ -662,28 +662,13 @@ nvim_echo({chunks}, {history}, {opts})                           *nvim_echo()*
                    `hl_group` element can be omitted for no highlight.
       • {history}  if true, add to |message-history|.
       • {opts}     Optional parameters.
+                   • err: Print an error message for which `history` is always
+                     true and omitting `hl_group` uses |hl-ErrorMsg|.
```

Comment:
`history` is a required arg so it's strange to ignore it. Is the `history is always true` behavior a limitation of the internals? If not, it seems better to just let the `history` arg decide this.

Reply:
>  Is the history is always true behavior a limitation of the internals

Yes, that limitation could be lifted by also passing along `history`; so I guess we should do that?

Reply:
I don't see why not. Maybe it raises the question of "why are we bothering to add `err` flag"? I assume it still has a purpose :) Maybe this doc isn't fully communicating what its purpose is.

Reply:
> Maybe it raises the question of "why are we bothering to add err flag"? I assume it still has a purpose

Yes, the thought crossed my mind. There is still some logic in `emsg_multiline` that applies to "emitting a message as an error". If nothing else, the message kind is appropriately conveyed to ext_messages. Not sure if all the other logic only applies to Vimscript.

Reply:
Sounds like it has a purpose then.